src/appConfigs/schema.ts

```diff
@@ -14,6 +17,20 @@ export const typeDefs = gql`
     toUid: ID!
   }
 
+  type ContactWith {
+    uid: ID!
+    met_on: String!
+    risk: String!
+  }
+
+  type ContactWithList {
+    metfriendon: String!
```

Comment:
@kavyvetri same again, `DateTime!` type

Reply:
Ignore this for now. not used this yet. will use later.